### DISCUSSION TITLE
Error in communication with SWARFARM

### DISCUSSION THREAD

**Discussion by LBarolin** [Xzandro/sw-exporter#419] [Q&A] [answered]
Hey,

I have a little (big) problem with SWEX lately. I was on v0.0.35 (yeah I didn't check for updates but it was working anyway so...).

My cert expired, so I regenerated it (meanwhile I updated SWEX). But I still got some errors like :

- Error while connecting to SWARFARM
- Error submitting log event to SWARFARM.
- Error while synchronizing profile with SWARFARM.

Did I do something wrong ? Is there a real problem with Swarfarm this days ?

**Comment by Xzandro** [comment, marked as answer]
This is / was a bug in the upstream package electron: https://github.com/electron/electron/issues/31212

Need to wait, until they release a new version and make it available on npm.

**Reply by LBarolin** [reply]
Okay ty for the fast response, I'll wait :)
